docs/catalog/sinks/RobustaUI.rst

```diff
@@ -0,0 +1,44 @@
+Robusta UI
+#################
+
+Robusta can send Playbooks results to Robusta UI.
+
+To configure Robusta UI, use our CLI to generate a token. Run:
+
+.. code-block:: cb-robusta-ui-sink-generate-token
+
+    robusta integrations ui
+
+
+Configuring the Robusta UI sink
+------------------------------------------------
+
+.. admonition:: Add this to your generated_values.yaml
+
+    .. code-block:: cb-robusta-ui-sink-config
+
+        sinksConfig:
+            - robusta_sink:
+                name: robusta_ui_sink
+                token: <your-token>
+                ttl_hours: 4380
+
+.. note::
+
+    | **Unactive** Robusta clusters have a default 6 months **data retention** span (4380 hours).
+    | For Test or ephemeral clusters you can use a custom data retention span using ``ttl_hours = <number-time-span>``.
```

Comment:
Don't capitalize `Test`.

I would phrase this a little differently in terms of the use case. 

`To prevent short-lived clusters from fililng up the UI, you can remove clusters that haven't communicated for more than a few hours by setting`...